packages/sanity/src/structure/components/paneItem/PaneItemPreview.tsx

```diff
@@ -59,12 +59,13 @@ export function PaneItemPreview(props: PaneItemPreviewProps) {
       </Flex>
     </TooltipDelayGroupProvider>
   )
-
+  const previewTitle = (title || draft?.title || published?.title) as string | null
   const tooltip = <DocumentStatus draft={draft} published={published} />
 
   return (
     <SanityDefaultPreview
       {...getPreviewValueWithFallback({value, draft, published})}
+      title={previewTitle}
```

Comment:
Will appreciate thoughts here, this allows the PaneItemPreview to define it's own title if the it doesn't receive a `title` property.  see https://github.com/sanity-io/sanity/pull/6889/files#diff-e4eac7882bb83a0d1cea009699c82f74030e040abb07bdc7128c73b761ea192bR40
Previously, we needed to use twice the `getPreviewStateObservable` function to get the title and then pass it down to this component. Given this component is already using it, I think it makes sense to allow the component define the title if it has not been received in the props.
It allowed for this change: https://github.com/sanity-io/sanity/pull/6889/commits/8bb969aca30e7be006e535159086702fea2a329c